File: pset6/mario/more/mario.py
from cs50 import get_string, get_int


def main():
    h = get_height()
    print_stairs(h)


def get_height():
    while(True):
        height = get_int("Height: ")
        if 0 < height < 9:
            break
    return height


def print_stairs(h):
    for i in range(h):
        for k in range(h - (i + 1), 0, -1):
            print(' ', end="")
        for j in range((h - i - 1), h):
            print('#', end="")
        print('  ', end="")
        for l in range((h - i - 1), h):
            print("#", end="")
        print()


main()
        

File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    // request for hieght in interger value only
    do
    {
        height = get_int("Enter Height: ");
    }
    //check for int value is between 1 and 8
    while (height < 1 || height > 8);
    
    for (int i = 1; i <= height; i++)
    {
        //space before left stairs starts
        for (int k = height - i; k > 0; k--)
        {
            printf(" ");
        }
        // left stairs
        for (int j = 1; j <= i; j++)
        {
            printf("#");
        }
        //gap between stairs
        printf("  ");
        
        //right stairs
        for (int m = 1; m <= i; m++)
        {
            printf("#");
        }
        printf("\n");
    }
}
